Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -142,10 +136,7 @@ private void OnLoaded(object sender, RoutedEventArgs e)
 
             _narrowState = GetTemplateChild(NarrowState) as VisualState;
 
-            SetVisualState(_stateGroup.CurrentState, true);
-            SetBackButtonVisibility(_stateGroup.CurrentState);
-
-            UpdateViewState();
+            UpdateView(true);
         }
 
         private void OnUnloaded(object sender, RoutedEventArgs e)
```

Comment:
Shouldn't the OnUnloaded method be extended with a method removing the BackButton? Right now, when opening the details view and than navigating to another page, the back button remains visible. I think the back should no longer be visable in that scenario. If the user navigates back and the viewState is still Details, at that point it can be made visible again.